pkg/mcs/tso/server/server.go

```diff
@@ -258,88 +247,5 @@ func (s *Server) GetTLSConfig() *grpcutil.TLSConfig {
 
 // CreateServerWrapper encapsulates the configuration/log/metrics initialization and create the server
 func CreateServerWrapper(cmd *cobra.Command, args []string) {
-	cmd.Flags().Parse(args)
```

Comment:
move the implementation to next pr to satisfy coverage check.

Reply:
coverage check is not necessary check

Reply:
I see